<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸检索demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="videoViewer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>750</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>1000</width>
            <height>750</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="MainWindow.qrc">:/bg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_detectFace">
            <property name="text">
             <string>检测人脸(D)
(如果已经是人脸，请勿点击)</string>
            </property>
            <property name="shortcut">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_open">
          <property name="text">
           <string>打开文件(F)</string>
          </property>
          <property name="shortcut">
           <string>F</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_camera">
          <property name="text">
           <string>打开摄像头(C)</string>
          </property>
          <property name="shortcut">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addToLib">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>添加到库(A)</string>
          </property>
          <property name="shortcut">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_retrieval">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>检索(SPACE)</string>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exit">
          <property name="text">
           <string>退出(ESC)</string>
          </property>
          <property name="shortcut">
           <string>Esc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置与管理</string>
    </property>
    <addaction name="action_threshold"/>
    <addaction name="action_manageFeatureLib"/>
    <addaction name="action_lookupRecord"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_threshold">
   <property name="text">
    <string>设置阈值</string>
   </property>
  </action>
  <action name="action_manageFeatureLib">
   <property name="text">
    <string>管理特征库</string>
   </property>
   <property name="iconText">
    <string>管理特征库</string>
   </property>
   <property name="toolTip">
    <string>管理特征库</string>
   </property>
  </action>
  <action name="action_lookupRecord">
   <property name="text">
    <string>查看记录</string>
   </property>
   <property name="iconText">
    <string>查看记录</string>
   </property>
   <property name="toolTip">
    <string>查看记录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
